src/main/java/spireMapOverhaul/zones/thefog/events/StandingStones.java

```diff
@@ -0,0 +1,88 @@
+package spireMapOverhaul.zones.thefog.events;
+
+import com.badlogic.gdx.math.MathUtils;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.AbstractImageEvent;
+import com.megacrit.cardcrawl.localization.EventStrings;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import spireMapOverhaul.rewards.AnyColorCardReward;
+import spireMapOverhaul.zones.thefog.relics.StoneFragment;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+import static spireMapOverhaul.SpireAnniversary6Mod.makeImagePath;
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class StandingStones extends AbstractImageEvent {
+    public static final String ID = makeID("StandingStones");
+
+    private static final EventStrings eventStrings = CardCrawlGame.languagePack.getEventString(ID);
+
+    public static final String NAME = eventStrings.NAME;
+    public static final String[] DESCRIPTIONS = eventStrings.DESCRIPTIONS;
+    public static final String[] OPTIONS = eventStrings.OPTIONS;
+
+    private int screenNum = 0;
+
+    private static final float HP_HEAL_PERCENT = 0.33F;
+    private static final float A15_HP_HEAL_PERCENT = 0.2F;
+    private static final float HP_DAMAGE_PERCENT = 0.15F;
+    private static final float A15_HP_DAMAGE_PERCENT = 0.2F;
+
+    private final int healAmt;
+    private final int damageAmt;
+
+    public StandingStones() {
+        super(NAME, DESCRIPTIONS[0], makeImagePath("events/TheFog/StandingStones.png"));
+        this.noCardsInRewards = true;
+
+        if (AbstractDungeon.ascensionLevel >= 15) {
+            this.healAmt = MathUtils.round(adp().maxHealth * A15_HP_HEAL_PERCENT);
+            this.damageAmt = MathUtils.round(adp().maxHealth * A15_HP_DAMAGE_PERCENT);
+        } else {
+            this.healAmt = MathUtils.round(adp().maxHealth * HP_HEAL_PERCENT);
+            this.damageAmt = MathUtils.round(adp().maxHealth * HP_DAMAGE_PERCENT);
+        }
+
+        this.imageEventText.setDialogOption(OPTIONS[0] + this.healAmt + OPTIONS[1]);
+        this.imageEventText.setDialogOption(OPTIONS[2]);
+        this.imageEventText.setDialogOption(OPTIONS[3] + this.damageAmt + OPTIONS[4]);
```

Comment:
Generally when the player is getting a specific fixed relic, the event will preview it on hover and sometimes say what it is in the text. See e.g. Golden Idol. For a modded event, we should at least have the preview on hover; if this isn't transparent about what the player is getting, people are less likely to engage with the content. So let's do that here.

You can provide a copy of the relic as the second parameter to `setDialogOption` to get the preview on hover. Up to you if you want to include the relic name in the text for the option.